tests/test_python.py

```diff
@@ -135,6 +135,27 @@ def test_tox(auto_container_per_test):
     )
 
 
+@pytest.mark.xfail(
+    OS_VERSION in ("15.6", "15.7"),
+    reason="https://bugzilla.suse.com/show_bug.cgi?id=1232072",
+)
```

Comment:
The xfail doesn't appear to be valid for SP6

Reply:
it is valid for SP6, that's where the bug is. the issue is that it is only valid for the python 3.6 version not the python 3.11 version of tox. 

I don't know how to express that in an mark.xfail, so for now I've removed the condition. the underlying bug is in progress of being fixed, so with some luck it is released before the next review of this PR happens.